libraries/stdlib/src/kotlin/time/Duration.kt

```diff
@@ -42,6 +42,119 @@ public inline class Duration internal constructor(internal val value: Double) :
         /** Converts the given time duration [value] expressed in the specified [sourceUnit] into the specified [targetUnit]. */
```

Comment:
Can't figure out how to leave a comment to unchanged lines, leaving here, KDoc to Duration class:

>To construct a duration use either the extension function toDuration, or the extension properties hours, minutes, seconds, and so on, available on Int, Long, and Double numeric types.

We should recommend using factories instead of extensions